NawasNaziru/Banka
# #165399662 use es6 features through out  the app

What does this PR do?
  Have es6 features like modules and arrow functions used through out.

Description of task to be completed?
  Replace old javascript module.exports and functions with es6 modules and arrow functions.

How should this be manually tested?
  Re-run the application testfile and test the API endpoints with nodemon to make sure nothing breaks.

Any background context you want to provide?
There is the need for use of latest language features to make future integration with tools easy and smooth.

What are the relevant pivotal tracker stories?
#165399662
